projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss

```diff
@@ -40,36 +40,38 @@
   @include mixins.generate-typography-token(
     'SECONDARY-13-RG-EXP',
     (
-      font-size: dropdown-variables.$clr-dropdown-item-font-size,
       font-weight: dropdown-variables.$clr-dropdown-item-font-weight,
-      line-height: dropdown-variables.$clr-dropdown-item-line-height,
-      letter-spacing: dropdown-variables.$clr-dropdown-item-letter-spacing,
     )
   );
 
   color: dropdown-variables.$clr-dropdown-item-color;
-  background: transparent;
+  background-color: dropdown-variables.$clr-dropdown-item-bg-color;
```

Comment:
Revert to transparent as fixed in figma.

Reply:
Transparent background is set through `--clr-dropdown-item-bg-color` and `--clr-dropdown-item-disabled-bg-color` properties in commit 9fd09fd